File: 27-remove-element/27-remove-element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        idx = 1
        while val in nums:
            if nums[idx-1] == val:
                nums.remove(nums[idx-1])
                idx-=1
            idx+=1
        return len(nums)

File: 27-remove-element/27-remove-element.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        if(!nums.size()){return 0;}
        int s = 0, e = nums.size();
        while(s<=e-1){
            if(nums[s] == val && nums[e-1] != val){
                swap(nums[s], nums[e-1]);
                e -= 1;
            }
            else if(nums[e-1] == val){e-=1; continue;}
            s+=1;
        }
        // if(s == 0 && nums[s] == val){return s;}
        return e;
    }
};